### PR TITLE
Rules update for auditor selection timeline

### PR DESCRIPTION
**Pull request by mrasquinha-g**
As discussed in the working group on 2/28. Updating the rules to restrict the time for auditor selection.

### CODE REVIEW

**Comment by DilipSequeira** [comment]
What do we want to do in the case where the committee fail to find an auditor?

**Comment by DilipSequeira** [comment]
Also, it might be better to say "28 days", just because the existing timeline is framed in units of days.

**Comment by mrasquinha-g** [comment]
> 
As discussed in the WG meeting the results review committee will have a list of potential auditors to pick from, minimizing the chances of such a situtation.

In the case that "no auditor" can be identified within 28 days and the audit process exceeds three months. The submitter can request for an audit to be cancelled.

**Comment by mrasquinha-g** [comment]
Updated.

File: inference_rules.adoc [hunk 1]
```diff
@@ -123,6 +123,7 @@ If a submission failed an audit that was delayed past publication, then any publ
 
 MLCommons shall retain a library of past audit reports and send copies to MLCommons members, auditors, and potential auditors by request. Audit reports will not be further distributed without permission from the audited submitter.
 
+An audit is expected to be completed within a 90 day period. Audits failing to meet this timeline will be nullified by request of the auditee.
```
Comment (DilipSequeira) [inline inference_rules.adoc:126]: Once the auditor is chosen, the submitter has a great deal of control over the timeline - they're primarily responsible for constructing an NDA the auditor can sign, and primarily responsible for providing the information the auditor needs to produce the audit. So a submitter could run out the clock if they so choose.

My understanding of what was discussed in the meeting was that the submitter can request to the WG that the audit be nullified (and then presumably they'd bring good reasons why it didn't happen.)

Comment (mrasquinha-g) [reply inference_rules.adoc]: That makes sense. Updated to clarify that the submitter can request for a invalidation and the final decision rests with the Working Group.

File: inference_rules.adoc [hunk 2]
```diff
@@ -123,7 +123,8 @@ If a submission failed an audit that was delayed past publication, then any publ
 
 MLCommons shall retain a library of past audit reports and send copies to MLCommons members, auditors, and potential auditors by request. Audit reports will not be further distributed without permission from the audited submitter.
 
-An audit is expected to be completed within a 90 day period. Audits failing to meet this timeline will be nullified by request of the auditee.
+An audit is expected to be completed within a 90 day period. Audits failing to meet this timeline can be requested to be invalidated by the auditee. The final decision to accept such a request will be determined by the Working Group.
```
Comment (DilipSequeira) [inline inference_rules.adoc:126]:
```suggestion
An audit is expected to be completed within a 90 day period. Audits failing to meet this timeline can be requested to be invalidated by the auditee. The final decision to accept such a request will be taken by the Working Group.
```

**Review by rnaidu02 — APPROVED** [review]

**Pull request merged**